README.md

```diff
@@ -168,29 +168,32 @@ service studentData on studentServiceListener {
 
         if (payloadJson is json) {
             //Converting the payload to Student type.
-
             Student|error studentDetails = Student.convert(payloadJson);
 
             if (studentDetails is Student) {
                 io:println(studentDetails);
                 // Calling the function insertData to update database.
                 json returnValue = insertData(untaint studentDetails.name, untaint studentDetails.age, untaint studentDetails.mobNo, untaint studentDetails.address);
                 response.setJsonPayload(untaint returnValue);
+            } else {
+                log:printError("Error in converting JSON payload to student type", err = studentDetails);
             }
+        } else {
+            log:printError("Error obtaining the JSON payload", err = payloadJson);
         }
-
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default ootb system span.
+        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the traces. These tags are added to the default system span.